components/server/src/bitbucket-server/bitbucket-server-api.spec.ts

```diff
@@ -174,35 +174,6 @@ class TestBitbucketServerApi {
         //     ✓ test_getRepos_searchString_wildcards_are_not_supported (172ms)
     }
 
-    @test async test_getRepos_searchString_single_char_is_ignored() {
```

Comment:
This didn't really make sense to me as a test, because single chars are not ignored with BBS.